scripts/scriptutils/part_retrieval.py

```diff
@@ -16,7 +16,8 @@
 from sbol_utilities.excel_to_sbol import BASIC_PARTS_COLLECTION
 from .directories import EXPORT_DIRECTORY, SBOL_EXPORT_NAME
 from .package_specification import package_stem
-from sbol_utilities.conversion import convert2to3, convert_from_fasta, convert_from_genbank
+from .conversions import convert2to3
```

Comment:
Why is this coming from .conversions rather than sbol_utilities now?

Reply:
Most likely slipped through, but to clarify:

We are switching to sbol-utilities for the conversions, so all .conversions functions are needed, right?

Reply:
There should be only one function left in .conversions, which is "convert_package_sbol2_files"

However, that function uses convert2to3, so what's happening here is an import of an import, which works but isn't good form.